modules/mobile/app/controllers/mobile/v0/profile_base_controller.rb

```diff
@@ -22,6 +24,53 @@ def render_transaction_to_json(transaction)
       def service
         Mobile::V0::Profile::SyncUpdateService.new(@current_user)
       end
+
+      def log_sso_info
+        action = request.controller_instance.controller_path.classify.to_s
+        action += 'Controller#'
+        action += request.parameters['action'].to_s
+
+        Rails.logger.warn(
+          "#{action} request completed", sso_logging_info
+        )
+      end
+
+      def sso_logging_info
+        { user_uuid: @current_user&.uuid,
+          sso_cookie_contents: sso_cookie_content,
+          request_host: request.host }
+      end
```

Comment:
could this sso_logging info be included from elsewhere in vets-api or did you have to do some customization to get it to work in mobile?

Reply:
It needs to be in a controller and the module from vets-api that already contains it overwrites things we use in mobile, so we can't just include the module